### DISCUSSION TITLE
Does GHA support building from source so as to run on other platforms?

### DISCUSSION THREAD

**Discussion by fengyuentau** [community/community#18546] [Actions]
Hello! I have some other platforms, such as RISC-V, and want to have some tests on pull requests on those platforms which are not officially supported. Can GHA be built from source and used in such platforms? I did a quick search in https://github.com/actions/runner and did not find a guide how to do that.

**Comment by doguedogue** [comment]
Hope this help you 🚀

[open-source-custom-github-actions-runners-with-google-cloud-and-terraform-antmicro/](https://riscv.org/blog/2021/08/open-source-custom-github-actions-runners-with-google-cloud-and-terraform-antmicro/)
